mmcv/ops/iou3d.py

```diff
@@ -65,25 +164,28 @@ def nms_bev(boxes, scores, thresh, pre_max_size=None, post_max_size=None):
 
 
 def nms_normal_bev(boxes, scores, thresh):
-    """Normal NMS function GPU implementation (for BEV boxes). The overlap of
+    """Normal NMS function GPU implementation (for BEV boxes).
+
+    The overlap of
     two boxes for IoU calculation is defined as the exact overlapping area of
     the two boxes WITH their yaw angle set to 0.
-
     Args:
         boxes (torch.Tensor): Input boxes with shape (N, 5).
         scores (torch.Tensor): Scores of predicted boxes with shape (N).
         thresh (float): Overlap threshold of NMS.
-
     Returns:
         torch.Tensor: Remaining indices with scores in descending order.
     """
+    warnings.warn(
+        '`iou3d.nms_normal_bev` is deprecated and will be removed in'
+        ' the future. Please, use `nms.nms`.', DeprecationWarning)
     assert boxes.shape[1] == 5, 'Input boxes shape should be [N, 5]'
     order = scores.sort(0, descending=True)[1]
 
-    boxes = boxes[order].contiguous()
+    boxes = _xyxyr2xyzwlhr(boxes)[order].contiguous()
```

Comment:
Similar suggestion as that for boxes_iou_bev. We can simply use nms.nms here to simplify the code without breaking the compatibility.